internal/impl/gcp/output_bigquery.go

```diff
@@ -276,7 +283,9 @@ type gcpBigQueryOutput struct {
 
 	fieldDelimiterBytes []byte
 	csvHeaderBytes      []byte
-	newLineBytes        []byte
+	// if nil, then this is a format that we expect to be created downstream in a processor and each
```

Comment:
I might be misreading this, but are the processors not upstream?

Reply:
Lol 🙃